src/QsCompiler/DataStructures/Diagnostics.fs

```diff
@@ -612,6 +614,7 @@ type DiagnosticItem =
             | ErrorCode.MultipleEntryPoints                       -> "Invalid entry point. An entry point {0} already exists in {1}."
             | ErrorCode.InvalidEntryPointSpecialization           -> "Entry points cannot have any other specializations besides the default body."
             | ErrorCode.DuplicateEntryPointArgumentName           -> "Invalid name for entry point argument. A similar argument name is already in use."
+            | ErrorCode.EntryPointInLibrary                       -> "Invalid entry point. Only Q# command line applications can have entry points."
```

Comment:
```suggestion
            | ErrorCode.EntryPointInLibrary                       -> "Invalid entry point. Only executable Q# projects can have entry points."
```

Someone could write a custom driver that has a GUI. :)

Reply:
alright, done.